src/shogun/mathematics/linalg/GPUMemoryViennaCL.h

```diff
@@ -53,6 +54,11 @@ struct GPUMemoryViennaCL : public GPUMemoryBase<T>
 
 	typedef viennacl::backend::mem_handle VCLMemoryArray;
 	typedef viennacl::vector_base<T, std::size_t, std::ptrdiff_t> VCLVectorBase;
+#if VIENNACL_VERSION >= 10600
```

Comment:
The viennaCL docs should talk about this API change somewhere. Where do you know this from otherwise?
I just want a simple HTTP link in a source code comment here